include/Network/HTTP/HTTPException.php

```diff
@@ -0,0 +1,37 @@
+<?php
+/**
+ * Throwable exceptions to return HTTP status code
+ *
+ * This list of Exception has be extracted from
+ * here http://racksburg.com/choosing-an-http-status-code/
+ */
+namespace Friendica\Network\HTTP;
```

Comment:
STANDARDS: please insert a new line between the doxygen header and the namespace